src/google_apps_clj/google_sheets_v4.clj

```diff
@@ -64,8 +65,34 @@
       (.get spreadsheet-id)
       (.execute)))
 
+(defn get-sheet-titles
+  [service spreadsheet-id]
+  "returns a list of [sheet-title sheet-id] tuples. It seems the order reflects
+  the order of the tabs in google's interface, though I doubt this is anywhere
+  guaranteed."
+  (->> (get (get-sheet-info service spreadsheet-id) "sheets")
+       (map #(get % "properties"))
+       (map (juxt #(get % "title") #(get % "sheetId")))))
+
+(defn date-time
+  "using the given excel date formatting string, eg \"yyyy-mm-dd\", \"M/d/yyyy\" "
+  [date-time format-patterm]
```

Comment:
I'm not sure I'm against per-cell formatting. We do value-based formatting on our aging reports, and also I don't know if we want to enforce column-based tables at the library level. And I feel like the sets of formats is fixed and based on google, rather than something that can be extensible by clients. I.e. I would imagine adding just a bunch of plain-old functions/combinators that represent google's formatting.  But you know me I'm the least pro-protocol person here!